### ISSUE TITLE
create gwas or other genomics workflow example

### ISSUE THREAD

**Issue by joestubbs** [joestubbs/endofday#2]
Will likely also need the docker images.

**Comment by deardooley** [comment]
Images:
https://www.biostars.org/p/117401/

Primer:
https://www.biostars.org/p/76993/

**Comment by deardooley** [comment]
http://bioboxes.org/

**Comment by AEStapleton** [comment]
My iPlant students and I would happy to help with a gwas workflow.  Just let us know what you need, we are familiar with the apps and the known-truth test data.

**Comment by joestubbs** [comment]
That would be fantastic! The idea would be to create a yaml file that represented a real-world workflow with valid inputs and docker images. We have a set of docker images in the hub under the organization taccsciapps that has a growing collection of images for biology software including bwa, samtools, gatk, etc:

https://registry.hub.docker.com/search?q=taccsciapps

We also have an example of some exome analysis in this examples directory here:

https://github.com/joestubbs/endofday/blob/master/examples/exome_analysis.yml

which should be mostly correct, but it needs to be tested and debugged with real data. This example relies on picard tools which we don't have an image for yet.

If you already have a workflow you would like to implement let's just do that. We can help with any docker images that need to be created and any other issues you might encounter.

Thanks!

**Comment by AEStapleton** [comment]
We would love help with this!  What do you think about 'dockerizing' our Validate workflow, https://github.com/UNCW-iPlant/The-Validate-Project?  If this seems like too much, or if the fact that we've got a couple more versions to go before we have full functionality, then we can just start with a GWAS app and our simulated data files.  I'll talk with the gang tomorrow and we will make a list of questions for you.  Yours, Ann

On May 21, 2015, at 5:51 PM, joestubbs <notifications@github.com<mailto:notifications@github.com>> wrote:

That would be fantastic! The idea would be to create a yaml file that represented a real-world workflow with valid inputs and docker images. We have a set of docker images in the hub under the organization taccsciapps that has a growing collection of images for biology software including bwa, samtools, gatk, etc:

https://registry.hub.docker.com/search?q=taccsciapps

We also have an example of some exome analysis in this examples directory here:

https://github.com/joestubbs/endofday/blob/master/examples/exome_analysis.yml

which should be mostly correct, but it needs to be tested and debugged with real data. This example relies on picard tools which we don't have an image for yet.

If you already have a workflow you would like to implement let's just do that. We can help with any docker images that need to be created and any other issues you might encounter.

Thanks!

�

**Comment by joestubbs** [comment]
I think it would be great to dockerize your Validate workflow! I have limited experience with bioinfomartics (I've learned just enough to do some consulting here and there for real biologists) but would be glad to help in any way I can. Let me take some time tomorrow to dig into your documentation and let's check back in over the next day or so. Talk soon!

**Comment by AEStapleton** [comment]
And our feature list is still in trello, not yet in github, let my know if you'd like to be added to that board.  We are still deciding on the order of features for each release.  Yours, Ann

On May 21, 2015, at 10:41 PM, joestubbs <notifications@github.com<mailto:notifications@github.com>> wrote:

I think it would be great to dockerize your Validate workflow! I have limited experience with bioinfomartics (I've learned just enough to do some consulting here and there for real biologists) but would be glad to help in any way I can. Let me take some time tomorrow to dig into your documentation and let's check back in over the next day or so. Talk soon!

�
